README.md

```diff
@@ -44,18 +44,20 @@ module.exports = {
     default: 'node index.js',
     lint: 'eslint .',
     test: {
-      // learn more about Jest here: https://kcd.im/egghead-jest
+      // learn more about Jest here: https://facebook.github.io/jest
       default: 'jest',
       watch: {
         script: 'jest --watch',
         description: 'run in the amazingly intelligent Jest watch mode'
       }
     },
     build: {
+      // learn more about Webpack here: https://webpack.js.org/
       default: 'webpack',
       prod: 'webpack -p',
     },
-    validate: 'nps --parallel lint,test,build',
+    // learn more about concurrently here: https://npm.im/concurrently
+    validate: 'concurrently "nps lint" "nps test" "nps build"',
```

Comment:
I don't think that we could support that syntax unfortunately. Because concurrently will spawn a process with the command `nps lint test build`. If you look at the additions in `EXAMPLES.md` you'll see that a function can be used to simplify things further. Perhaps we could use that example in the README instead?